libopenarc/arc-canon.c

```diff
@@ -2135,16 +2135,20 @@ arc_parse_canon_t(unsigned char *tag, arc_canon_t *hdr_canon,
 		return ARC_STAT_INVALID;
 
 	token = strtok_r(tag, "/", &last);
+	if (token == NULL)
+		return ARC_STAT_INVALID;
+		
```

Comment:
This cannot happen. There will always be an initial token, because the case where the string is empty returns before this call.